src/test/java/uk/gov/hmcts/befta/dse/ccd/definition/converter/EnvironmentURLUtilsTest.java

```diff
@@ -115,7 +118,8 @@ void setup() throws IOException {
     @Test
     void unsupportedSheetNamesShouldNotModifyURLs() throws Exception{
         JsonNode nullJsonNode = objectMapper.nullNode();
-        JsonNode modifiedJsonNode = EnvironmentURLUtils.updateCallBackURLs(nullJsonNode, "AFileNameThatIsNotCaseEvent");
+        JsonNode modifiedJsonNode = EnvironmentURLUtils.updateCallBackURLs(nullJsonNode, "AFileNameThatIsNotCaseEvent",
```

Comment:
It would be nice to squeeze in at least one extra test where the Environment rename occurs.

Reply:
Updated 2 tests to challenge the env value replacement in the final url value

Reply:
Done.